Source/WebCore/platform/graphics/ComplexTextController.h

```diff
@@ -169,6 +172,7 @@ class ComplexTextController {
     Vector<FloatSize, 256> m_adjustedBaseAdvances;
     Vector<FloatPoint, 256> m_glyphOrigins;
     Vector<CGGlyph, 256> m_adjustedGlyphs;
+    Vector<float, 256> m_textAutoSpaceSpacings;
```

Comment:
Why `256` ?

Reply:
All our parallel vectors at `ComplexTextRun` are initialized with a 256 capacity, this will have the same size as the others as this is another parallel vector. We might consider recalculating what is the optimal initial capacity but not here.

```
    Vector<FloatSize, 256> m_adjustedBaseAdvances;
    Vector<FloatPoint, 256> m_glyphOrigins;
    Vector<CGGlyph, 256> m_adjustedGlyphs;
```

Reply:
This could use a follow-up refactoring patch to change this magic number to a template parameter maybe